src/test/java/com/snack/news/controller/NewsControllerTest.java

```diff
@@ -43,8 +44,8 @@
 
 	private MockMvc mockMvc;
 
-	@Before
-	public void setup() {
+	@BeforeEach
+	void setup() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [You may have misspelled a JUnit framework method (setUp or tearDown)](https://app.codacy.com/manual/snack-news/Snack-BE/pullRequest?prid=4863700)